Readme.md

```diff
@@ -22,7 +22,7 @@ Aerial is completely open source, so feel free to contribute to its development!
 
 ## Installation
 
-1. [Click here to Download](https://github.com/esetnik/Aerial/releases/download/v1.3/Aerial.saver.zip)
+1. [Click here to Download](https://github.com/glouel/Aerial/releases/download/v1.4a1/Aerial.saver.zip)
```

Comment:
@glouel we need to be careful here. If we want this to actually get merged into JohnCoates/Aerial then this shouldn't point to the forked repo.

Reply:
Is there any hope of any of these changes being pulled back in to this repo here? It doesn't look like it has accepted any real code changes in a long time. Maybe it's time to settle around a new fork?

Reply:
I've seen the same thing. I just opened https://github.com/JohnCoates/Aerial/issues/499 to formally ask for more active project maintenance. I think it's best to keep it under the original repo and have active maintainers take over the project. It's already got 11,795 stars and lots of public links to the project on blogs, media, etc.